csharp/Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /usr/src/app
 
 # copy csproj and restore as distinct layers
 COPY *.csproj .
+COPY Directory.Build.props .
```

Comment:
not sure it will work, docker root directoy is `csharp/carter` not `csharp`

Reply:
why its set to csharp/carter? hmm, how do I achieve this now?

Reply:
I did ../Directory.build.props but still does not work. any idea?

Reply:
the easyiest way to do the trick @ShreyasJejurkar is, imho, to use symlinks (create main file is `csharp`, and create symlinks in all frameworks directory)